### PR TITLE
[Outlook] (legacy) Refresh docs and update snippet mappings

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[7aaf1a5](https://github.com/OfficeDev/office-js-docs-reference/pull/325/commits/7aaf1a5794c2dab42fa65335b05c415fa35d1f17)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[7aaf1a5](https://github.com/OfficeDev/office-js-docs-reference/pull/325/commits/7aaf1a5794c2dab42fa65335b05c415fa35d1f17)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Error occurred when restoring dependency repo from URL &#39;https://github.com/Microsoft/templates.docs.msft&#39; branch &#39;master&#39; into folder &#39;_themes&#39;: fatal: the remote end hung up unexpectedly
error: RPC failed; curl 18 transfer closed with outstanding read data remaining

 - **Line 165**: **[Error]** Initialize dependencies failed

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C15%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201902150059322689-325%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DClpdLQNVueHhUdh3vJBR9jJv8Z7IhstmSwGw50Vxwc%3D&st=2019-02-15T00%3A55%3A10Z&se=2019-03-18T01%3A00%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT Did you figure out why this build failed?

**Comment by o365devx** [comment]
OPS Build status updates of commit _[2e190a7](https://github.com/OfficeDev/office-js-docs-reference/pull/325/commits/2e190a7480ee04d69912762a33f721e36ef62a42)_: 

:clock10: **Preparing**: average preparing time is _1 min(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[2e190a7](https://github.com/OfficeDev/office-js-docs-reference/pull/325/commits/2e190a7480ee04d69912762a33f721e36ef62a42)_: 

:clock10: **Incremental building**: average incremental building time is _4 min(s) 50 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[2e190a7](https://github.com/OfficeDev/office-js-docs-reference/pull/325/commits/2e190a7480ee04d69912762a33f721e36ef62a42)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/outlook-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/code-snippets/outlook-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.body.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.body.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.enhancedlocation.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.enhancedlocation.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.internetheaders.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.internetheaders.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.item.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.item.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.location.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.location.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.notificationmessages.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.notificationmessages.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.organizer.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.organizer.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.recipients.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.recipients.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook/office.subject.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook/office.subject.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.appointmentcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.appointmentcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.appointmentform.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.appointmentform.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.appointmentread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.appointmentread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.attachmentdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.attachmentdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.body.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.body.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.contact.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.contact.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.customproperties.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.customproperties.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.diagnostics.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.diagnostics.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.emailaddressdetails.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.emailaddressdetails.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.emailuser.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.emailuser.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.entities.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.entities.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.item.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.item.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.itemcompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.itemcompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.itemread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.itemread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.localclienttime.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.localclienttime.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.location.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.location.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailbox.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailbox.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.attachmenttype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.attachmenttype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.entitytype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.entitytype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.itemtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.itemtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.recipienttype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.recipienttype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.responsetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.mailboxenums.responsetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.meetingsuggestion.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.meetingsuggestion.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.message.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.message.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.messagecompose.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.messageread.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.messageread.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.phonenumber.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.phonenumber.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.recipients.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.recipients.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.roamingsettings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.roamingsettings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.subject.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.subject.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.tasksuggestion.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.tasksuggestion.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.time.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.time.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_1/office.userprofile.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/get-latest/docs/docs-ref-autogen/outlook_1_1/office.userprofile.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C15%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201902152329400598-325%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2Qbzg6RTrxVRNBNE5gJbU1FdfDtCOIT2PMGTh8Jx4Vc%3D&st=2019-02-15T23%3A28%3A52Z&se=2019-03-18T23%3A33%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/code-snippets/outlook-snippets.yaml [hunk 1]
```diff
@@ -340,6 +340,30 @@ Office.AppointmentRead.dateTimeModified:
...
+    // Reply with a body, file attachment, item attachment, and a callback.
+    Office.context.mailbox.item.displayReplyAllForm(
+    {
+        'htmlBody' : 'hi',
+        'attachments' :
+        [
+            {
+                'type' : Office.MailboxEnums.AttachmentType.File,
+                'name' : 'squirrel.png',
+                'url' : 'http://i.imgur.com/sRgTlGR.jpg'
```
Comment (AlexJerabek) [inline docs/code-snippets/outlook-snippets.yaml]: What's the policy on non-MSFT links?

File: docs/docs-ref-autogen/outlook/office.appointmentcompose.yml [hunk 2]
```diff
@@ -1699,6 +1699,33 @@ items:
...
+
+
+      ```javascript
+
+      Office.context.mailbox.item.saveAsync(
+          function callback(result) {
+              // Process the result.
+          });
+
+      // The following is an example of the
```
Comment (AlexJerabek) [inline docs/docs-ref-autogen/outlook/office.appointmentcompose.yml]: The spacing on this comment seems a little weird. We could consolidate this to one or two lines.

**Comment by ElizabethSamuel-MSFT** [comment]
@AlexJerabek: Looks like something happened with the server yesterday. I pulled latest changes from d.ts but not seeing any issue now.

File: docs/docs-ref-autogen/outlook/office.appointmentread.yml [hunk 3]
```diff
@@ -371,6 +314,69 @@ items:
...
+
+      Office.context.mailbox.item.displayReplyAllForm(
+
+      {
+
+      'htmlBody' : 'hi'
+
+      });
+
+
```
Comment (AlexJerabek) [inline docs/docs-ref-autogen/outlook/office.appointmentread.yml]: There's a lot of extra blank lines in this sample.

File: docs/docs-ref-autogen/outlook/office.appointmentread.yml [hunk 4]
```diff
@@ -560,6 +511,67 @@ items:
...
+                  'name' : 'squirrel.png',
+                  'url' : 'http://i.imgur.com/sRgTlGR.jpg'
+              }
+          ]
+      });
+
+
+      // Reply with a body and an item attachment.
+
+      Office.context.mailbox.item.displayReplyForm(
```
Comment (AlexJerabek) [inline docs/docs-ref-autogen/outlook/office.appointmentread.yml]: Same extra blank lines comment.

Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/outlook/office.appointmentread.yml]: @AlexJerabek: It just looks spread out in the .yml file but on the site it looks like this:
image

File: docs/code-snippets/outlook-snippets.yaml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/code-snippets/outlook-snippets.yaml]: I'll dig into this and make a change subsequently if need be.

**Pull request merged**